v2/kafka-to-pubsub/src/main/resources/kafka_to_pubsub_metadata.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "Kafka to Pub/Sub",
+  "description": "Streaming pipeline. Ingests JSON-encoded messages from Kafka to a Pub/Sub topic",
```

Comment:
Would call as UTF8 encoded messages as pipeline uses String Deserializer only and never checks if value is a valid json or not.

Reply:
Fixed the description.